datafusion/functions-aggregate/src/median.rs

```diff
@@ -310,6 +310,18 @@ impl<T: ArrowNumericType> Accumulator for DistinctMedianAccumulator<T> {
     }
 }
 
+/// Get maximum entry in the slice,
+fn slice_max<T>(array: &[T::Native]) -> Option<T::Native>
+where
+    T: ArrowPrimitiveType,
+    T::Native: PartialOrd, // Ensure the type supports PartialOrd for comparison
+{
```

Comment:
As `unwrap` is used, the code could be updated / simplified to work on arrays of size >= 1

Reply:
That makes sense to me, updated the implementation such that function doesn't return option in [0303c9](https://github.com/apache/datafusion/pull/13554/commits/0303c965466c3b443f85e799fc3d8392877dbd20)